checks.d/openstack.py

```diff
@@ -167,7 +192,11 @@ def get_stats_for_single_network(self, network_id):
         resp = self._make_request_with_auth_fallback(url, headers)
 
         net_details = resp.json()
-        service_check_tags = ['network:{0}'.format(network_id)]
+        service_check_tags = [
+            'network:{0}'.format(network_id),
+            'network_name:{0}'.format(net_details.get('network', {}).get('name')),
+            'tenant_id:{0}'.format(net_details.get('network', {}).get('tenant_id'))
```

Comment:
If `network_name` or `tenant_id` is missing a significant amount of time, we should maybe check beforehand that the metadata we need exists, rather than adding tags like`network_name:None`.